prime-router/settings/prod/0034-hhsprotect.yml

```diff
@@ -0,0 +1,32 @@
+# Run ./prime receiver set --input ./settings/prod/0030-hhsprotect.yml --env prod
+# The change is additions to the juris filter to allow Waters data to flow to hhsprotect
+# Note that the processing_mode_code filter is still set to allow both "T" and "P" data, for now, in Prod.
+---
+name: "elr"
+organizationName: "hhsprotect"
+topic: "covid-19"
+translation: !<CUSTOM>
+  schemaName: "hhsprotect/hhsprotect-covid-19"
+  format: "CSV"
+  defaults: {}
+  nameFormat: "STANDARD"
+  receivingOrganization: null
+  type: "CUSTOM"
+jurisdictionalFilter:
+- "matches(sender_id, SafeHealth,ImageMover,Cue,InBios,Strac,.*CueHlth.*,.*ImageMover.*)"
```

Comment:
`.*CueHlth.*,.*ImageMover.*` means that any data with sender_id that contains `CueHlth` or `ImageMover` will now go to HHSProtect.  This should include almost all data coming from Waters.   (SafeHealth is already dual sending)